packages/quantic/cypress/integration/facets/numeric-facet/numeric-facet-expectations.ts

```diff
@@ -0,0 +1,100 @@
+import {InterceptAliases} from '../../../page-objects/search';
+import {should} from '../../common-selectors';
+import {SearchExpectations} from '../../search-expectations';
+import {
+  baseFacetExpectations,
+  facetWithValuesExpectations,
+} from '../facet-common-expectations';
+import {field} from './numeric-facet-actions';
+import {
+  AllFacetSelectors,
+  NumericFacetSelectors,
+} from './numeric-facet-selectors';
+
+const formatDefaultNumericFacetValue = (value: string) => {
+  value = value.trim();
+  const valueSeparator = '-';
+  const splitByValue = ` ${valueSeparator} `;
+  const start = value.split(splitByValue)[0].replace(/,/g, '');
```

Comment:
This replacement might work when locale is `en-US` but won't work with other locales like `fr-CA`. It appears we can't reliably parse a localized numerical value in JavaScript. Adding a custom attribute with the raw numerical value on the facet value element would make it easier to validate.